snowflake_data_profiler/profiling/profiler.py

```diff
@@ -76,7 +86,8 @@ def get_profile_results(data):
 
 
 def do_profile():
```

Comment:
Following my own advice, this is a bad function name. I should have used something else. How could we improve this?